onnx/onnx-ml.proto3

```diff
@@ -173,9 +179,13 @@ message NodeProto {
 // a parameterized computation graph against a set of named operators that are
 // defined independently from the graph.
 message ModelProto {
+  // 'ONXF' (reversed as 46 58 4E 4F) 0x46584E4F
```

Comment:
Why ONXF instead of ONNX?

Reply:
Keeping every byte distinct - just good practice to figure out byte ordering problems and serializers and whatnot. Not a particular problem when we all use protobuf, but good general practice.